ansible/main.yml

```diff
@@ -1,4 +1,41 @@
 ---
+
+- name: Project Bootstrap
+  hosts: all
+  tasks:
+    - name: Get User Token
+      uri:
+        url: "https://{{ clusterhost }}/oauth/authorize?response_type=token&client_id=openshift-challenging-client"
+        user: "{{ openshift_username }}"
+        password: "{{ openshift_password }}"
+        validate_certs: False
```

Comment:
I'd vote against disabling cert validation by default

Reply:
@codificat will add a variable for self-signed certs for both this and the insecure flag.